cms/templates/visibility_editor.html

```diff
@@ -27,8 +27,17 @@ <h4 class="title">${_('You have not set up any groups to manage visibility with.
         </div>
     % elif is_staff_locked:
         <div class="summary-message summary-message-warning visibility-summary-message">
-            <i class="icon icon-warning-sign"></i>
-            <p class="copy">${_('The Unit this component is contained in is hidden from students. Visibility settings here will be trumped by this.')}</p>
+            <i class="icon fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <p class="copy">
+                ## Translators: The "Warning:" text is meant to communicate the type of message being communicated and should always end with an extra space
```

Comment:
@cahrens I didn't write this one (I think DB helped @talbs). Since it got merged I'll make your suggested change on cohorted-courseware with my doc changes:

https://github.com/edx/edx-platform/pull/6498